### PR TITLE
updated Docker version information for Cloud

### LINKED ISSUE #3895: Update to Docker 17.06
File: [docker-cloud/cloud-swarm/index.md](https://docs.docker.com/docker-cloud/cloud-swarm/), CC @londoncalling

### PR DESCRIPTION
**Pull request by londoncalling**
### What's changed

- updated the Docker version referenced in Cloud swarm docs

### Reviewers

@ManoMarks @mstanleyjones @johndmulhausen 

### Related

Fixes #3895

### Notes

- I added an explicit reference to the current Docker version. We should automate this somehow so that as the version updates, the text changes. 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by ManoMarks** [comment]
Instead of Docker Engine 17.06, we should be referring to "Docker 17.06"

**Comment by londoncalling** [comment]
@ManoMarks I made some updates. Netlify is working now so here is a link to the Cloud Swarm page: https://deploy-preview-3896--docker-docs-vnext-engine.netlify.com/docker-cloud/cloud-swarm/

**Comment by ManoMarks** [comment]
LGTM

**Comment by londoncalling** [comment]
@mstanleyjones or @thaJeztah can you approve these changes so we can merge it?

File: engine/swarm/index.md [hunk 1]
```diff
@@ -4,17 +4,16 @@ keywords: docker, container, cluster, swarm
...
 ---
 
-To use Docker Engine in swarm mode, install the Docker Engine `v1.12.0` or
-later from the [Docker releases GitHub
-repository](https://github.com/moby/moby/releases). Alternatively, install
-the latest Docker for Mac or Docker for Windows Beta.
+To use Docker in swarm mode, install Docker `1.12.0` or later. Alternatively,
+install the latest Docker for Mac or Docker for Windows. Install instructions
+for all platforms are [here](https://docs.docker.com/engine/installation/). The
+most recent version is Docker 17.06.
```
Comment (thaJeztah) [inline engine/swarm/index.md]: I'd be cautious putting "latest version" in this document, as it easily gets outdated. Given that Docker versions older than 1.12 are no longer supported, perhaps we can get rid of this section.

File: docker-cloud/cloud-swarm/index.md [hunk 2]
```diff
@@ -8,15 +8,12 @@ title: Swarms in Docker Cloud (Beta)
...
-of Docker Engines running in v1.13
-[swarm mode](/engine/swarm/).
-
-With Beta Swarm Mode in Docker Cloud, you can
-provision swarms to popular cloud providers, or
-register existing swarms to Docker Cloud.
-Use your Docker ID to authenticate and securely access
-personal or team swarms.
+Docker Cloud now allows you to connect to clusters of Docker Engines running on
+Docker 17.06 [swarm mode](/engine/swarm/).
```
Comment (thaJeztah) [inline docker-cloud/cloud-swarm/index.md]: Does it only support 17.06, or also (e.g.) 17.03-EE?

File: engine/swarm/index.md (hunk 1 above)
Comment (londoncalling) [inline engine/swarm/index.md]: @thaJeztah I'm not sure what the issue would be with telling people, at time of installation, to install the most up-to-date version of Docker for Mac or Windows. I think this section is needed here, first to make sure they know that only `1.12.0` version of Docker Engine is supported, that if they are on Docker for Mac or Windows, the most up-to-date version will take care of that and they don't need to worry about it, and finally, where to go on the off chance that they are using an earlier version of Docker Engine. 

The only thing that will be time sensitive about the above section is that I'll have to remember to change what "the most recent version .." is, and I was hoping we could figure out a way to automate that. For now, I'm happy to update it, and I think it's nice to have.

File: docker-cloud/cloud-swarm/index.md (hunk 2 above)
Comment (londoncalling) [inline docker-cloud/cloud-swarm/index.md]: I think I've improved this overall per all of your comments, and got us out of the need to specify all these versions. Let me know what you guys think.

File: engine/swarm/index.md [hunk 3]
```diff
@@ -4,18 +4,16 @@ keywords: docker, container, cluster, swarm
...
-Docker Engines called a *swarm*. Use the Docker CLI to create a swarm, deploy
+Current versions of Docker include *swarm mode* for natively managing a cluster
+of Docker Engines called a *swarm*. Use the Docker CLI to create a swarm, deploy
 application services to a swarm, and manage swarm behavior.
 
-
-If you’re using a Docker version prior to `v1.12.0`, see [Docker
-Swarm](/swarm).
+If you are using a Docker version prior to `1.12.0`, you can use [standalone
+swarm](/swarm/index.md), but we recommend a updating.
```
Comment (JimGalasyn) [inline engine/swarm/index.md]: Typo: "recommend **a** updating".

Comment (JimGalasyn) [inline engine/swarm/index.md]: Otherwise, lgtm!

**Pull request merged**